docs/UserGuide.md

```diff
@@ -319,6 +332,10 @@ Expected behaviour upon success:
 Expected behaviour upon failure:
 - Refer to [Appendix B](#6-2-appendix-b-miscellaneous-error-messages): Expected behaviour upon general failure.
 
+<br />
+
+> [Back to top](#wedlog-user-guide)
+
```

Comment:
Not sure if the extra spacing that comes with `<br/>` is necessary, but up to you! 👍 

Before:
<img width="510" alt="image" src="https://github.com/AY2324S1-CS2103T-F11-2/tp/assets/61283948/34c78b0d-4baf-4678-b606-288b4ae6557f">

After:
<img width="508" alt="image" src="https://github.com/AY2324S1-CS2103T-F11-2/tp/assets/61283948/1958abca-666e-4fb8-87da-c9a5c2ef010d">